spec/spec_helper.rb

```diff
@@ -1,4 +1,121 @@
 # frozen_string_literal: true
-
+require 'webmock/rspec'
 $LOAD_PATH.unshift File.expand_path('../../lib', __FILE__)
 require 'companies_house/client'
+require 'timecop'
+
+def notifications_of
+  notifications = []
+  subscriber = ActiveSupport::Notifications.subscribe do |*args|
+    notifications << ActiveSupport::Notifications::Event.new(*args)
+  end
+
+  yield
+  ActiveSupport::Notifications.unsubscribe(subscriber)
+  notifications
+end
+
+shared_context 'test credentials' do
+  let(:api_key) { 'el-psy-congroo' }
```

Comment:
!["el psy congroo". that's great. i've got that on a T-shirt.](https://res.cloudinary.com/teepublic/image/private/s--r2jlLWlx--/t_Preview/b_rgb:191919,c_limit,f_jpg,h_630,q_90,w_630/v1446160249/production/designs/64814_1.jpg)